decentraland/unity-renderer
# Add AltTester package

## What does this PR change?



...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch={branch_name}&{desired_url_params}
2. Check that the AltTester popup is loaded
3. Check that you are able to connect to the AltTester socket using the AltTester Desktop app or via a script. 

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
